kafka-3.8.yaml

```diff
@@ -70,6 +70,30 @@ subpackages:
           image: kafka
           version-path: 3.8/debian-12
       - runs: |
+<<<<<<< HEAD
+=======
```

Comment:
this and one more line below can be removed. Sorry if you're already on it.

Reply:
Resolved - sorry for the noise!